Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -28,10 +29,50 @@
 
 namespace ROS2
 {
+    bool CollidersMaker::GuessIsWheel(const AZStd::string& entityName)
```

Comment:
move out to Utils (could be used in vehicle dynamics), and will be for sure used in the Importer when adding VD components